src/hotspot/share/runtime/reflection.cpp

```diff
@@ -767,9 +767,9 @@ static Handle new_type(Symbol* signature, Klass* k, TRAPS) {
 
 oop Reflection::new_method(const methodHandle& method, bool for_constant_pool_access, TRAPS) {
   // Allow sun.reflect.ConstantPool to refer to <clinit> methods as java.lang.reflect.Methods.
```

Comment:
Not quite related, but it's jdk.internal.reflect.ConstantPool now :)

Reply:
Fixed.